primefaces/src/main/java/org/primefaces/model/timeline/TimelineModel.java

```diff
@@ -151,9 +152,17 @@ public void update(TimelineEvent<E> event) {
      * @param timelineUpdater TimelineUpdater instance to update the event in UI
      */
     public void update(TimelineEvent<E> event, TimelineUpdater timelineUpdater) {
-        if (timelineUpdater != null) {
-            // update UI
-            timelineUpdater.update(event);
+        int index = IntStream.range(0, events.size())
```

Comment:
Can you add a comment above this like `\\ GitHub #8577` so we don't remove this again in the future :)